spec/controllers/families_controller_spec.rb

```diff
@@ -25,4 +25,34 @@
       end
     end
   end
+
+  describe "DELETE /families/:id" do
+    context "When the family id exsist" do
+      it "destroys that family" do
+        family = create(:family)
+
+        expect { delete :destroy, params: { id: family.id } }.to change(Family, :count).by(-1)
+      end
+
+      it "flashes a notice message and render the family list" do
+        family = create(:family)
+
+        delete :destroy, params: { id: family.id }
+
+        expect(flash[:notice])
+        .to match(
+          I18n.t("flash.actions.destroy.notice", resource_name: "Family")
+        )
+        expect(response).to redirect_to families_path
+      end
+    end
+
+    context "When the family id doesn't exist" do
+      it "flashes an alert message and render the family list" do
```

Comment:
A more accurate spec description would be:
`it "raises record not found error"`

Reply:
Done with fixup in b232054